Error when building release - "Could not normalize path for file"

I'm getting the following error on Windows 7 when running `gradlew assembleRelease`:


    :onebusaway-android:lintVitalRelease FAILED

    FAILURE: Build failed with an exception.

    * What went wrong:

    Execution failed for task ':onebusaway-android:lintVitalRelease'.
    > Could not normalize path for file 'F:\Android_Studio\onebusaway-android\onebusaway-android\build\intermediates\mockable-Google Inc.:Google APIs:19.jar'.

Our `build.gradle` originally had:

    compileSdkVersion "Google Inc.:Google APIs:19"
    buildToolsVersion "19.1.0"

I changed this to:

    compileSdkVersion "Google Inc.:Google APIs:21"
    buildToolsVersion "21.1.2"

... and I no longer get the above error.

Looks like there is an open issue on AOSP ([148912](https://code.google.com/p/android/issues/detail?id=148912)) for this.

But, for now, we can update our build tools and compileSdkVersion to fix it.

--- a/.travis.yml
+++ b/.travis.yml
@@ -9,8 +9,8 @@ cache: false
 sudo: false
 env:
   global:
-    - ANDROID_API_LEVEL=17
-    - ANDROID_BUILD_TOOLS_VERSION=19.1.0
+    - ANDROID_API_LEVEL=21
+    - ANDROID_BUILD_TOOLS_VERSION=21.1.2
     - ANDROID_ABI=armeabi-v7a
 
 android:

--- a/onebusaway-android/build.gradle
+++ b/onebusaway-android/build.gradle
@@ -29,8 +29,8 @@ repositories {
 }
 
 android {
-    compileSdkVersion "Google Inc.:Google APIs:17"
-    buildToolsVersion "19.1.0"
+    compileSdkVersion "Google Inc.:Google APIs:21"
+    buildToolsVersion "21.1.2"
 
     defaultConfig {
         minSdkVersion 9

